db_management/hosting_changes.rst

```diff
@@ -0,0 +1,63 @@
+:banner: banners/hosting_changes.png
```

Comment:
This file doesn't seem to exist :p

Reply:
@AntoineVDV  it still doesn't. Should I add one for my PR to backport this doc down to 12.0?
@rim-odoo did you have a picture for this + is it okay for you to backport to 12.0?

Cheers :)

Reply:
yup

Reply:
I forgot to create/add a picture, and if you want to back port it to 12.0 you can (this part of the doc is not versioned)